include/net/lwm2m.h

```diff
@@ -150,6 +150,13 @@ struct lwm2m_ctx {
 	 */
 	bool use_dtls;
 
+	/**
+	 * Flag for indicate that Socket is connected.
```

Comment:
"Flag to indicate that the socket is connected. With queue mode, this will tell if there is a need to reconnect"

Reply:
OK.  I looking for that. Thanks